File: mobileBE/src/main/java/com/uth/mobileBE/dto/response/ReaderResponse.java
package com.uth.mobileBE.dto.response;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReaderResponse {
    private Long readerId;
    private String fullName;
    private String phone;
    private String barcode;
    private Boolean isStudent;
    private Boolean isBlocked;
    private LocalDateTime membershipExpiry;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: app/src/main/java/com/example/quanlythuvien/data/model/response/ReaderResponse.kt
package com.example.quanlythuvien.data.model.response

import java.math.BigDecimal

data class ReaderResponse(
    val readerId: Long,
    val fullName: String,
    val phone: String,
    val barcode: String,
    val isBlocked: Boolean,
    val membershipExpiry: String?,
    val createdAt: String?,
    val totalBorrowedBooks: Int,
    val totalReturnBook:Int,
    val totalOverdueBooks: Int,
    val  totalDebt: BigDecimal?
)